### ISSUE TITLE
Compiled with problems: node_modules_lodash_debounce_index_js

### ISSUE THREAD

**Issue by unsortedhashsets** [KaotoIO/kaoto-ui#1576]
### Describe the Bug

Hi, 

Faced problem with e2e test 02-editing_properties and was able to replicate manually:
(replicated with fixtures ChuckNorris.yaml and TimerKafka.yaml)

### Steps to Reproduce the Bug or Issue

1) Update some step configuration 
2) Try to update the configuration of some other step 
3) See the error

or use e2e test `02-editing_properties/editing.cy.js`

### Screenshots or Videos

Error:
![Screenshot from 2023-03-28 09-58-09](https://user-images.githubusercontent.com/46084942/228174071-d7622368-e7fd-4b8c-a3c2-080fd3a088a1.png)

Record:
[record.webm](https://user-images.githubusercontent.com/46084942/228174107-75575f6a-89ce-4a12-b4f5-35b02c0507bd.webm)

Stacktrace:
```
Compiled with problems:
×
ERROR
Cannot set properties of undefined (setting 'value')
TypeError: Cannot set properties of undefined (setting 'value')
    at http://localhost:1337/src_bootstrap_tsx.js:26717:53
    at Array.forEach (<anonymous>)
    at StepsService.updateStepParameters (http://localhost:1337/src_bootstrap_tsx.js:26715:39)
    at Object.<anonymous> (http://localhost:1337/src_bootstrap_tsx.js:25210:22)
    at invokeFunc (http://localhost:1337/node_modules_lodash_debounce_index_js.js:168:19)
    at trailingEdge (http://localhost:1337/node_modules_lodash_debounce_index_js.js:215:14)
    at timerExpired (http://localhost:1337/node_modules_lodash_debounce_index_js.js:203:14)
ERROR
Cannot set properties of undefined (setting 'value')
TypeError: Cannot set properties of undefined (setting 'value')
    at http://localhost:1337/src_bootstrap_tsx.js:26717:53
    at Array.forEach (<anonymous>)
    at StepsService.updateStepParameters (http://localhost:1337/src_bootstrap_tsx.js:26715:39)
    at Object.<anonymous> (http://localhost:1337/src_bootstrap_tsx.js:25210:22)
    at invokeFunc (http://localhost:1337/node_modules_lodash_debounce_index_js.js:168:19)
    at trailingEdge (http://localhost:1337/node_modules_lodash_debounce_index_js.js:215:14)
    at timerExpired (http://localhost:1337/node_modules_lodash_debounce_index_js.js:203:14)
```
### Platform

- OS: [RHEL-8.6]
- Browser: [Chrome]
- Version: [110]

**Comment by apupier** [comment]
@unsortedhashsets Can you provide the stacktrace as text too please? It allows the error stacktrace to be searchable

**Comment by apupier** [comment]
similar stack than https://github.com/KaotoIO/kaoto-ui/issues/1561

**Comment by unsortedhashsets** [comment]
> similar stack than #1561

Yes

**Comment by unsortedhashsets** [comment]
> @unsortedhashsets Can you provide the stacktrace as text too please? It allows the error stacktrace to be searchable

Here is double catch:

```
Compiled with problems:
×
ERROR
Cannot set properties of undefined (setting 'value')
TypeError: Cannot set properties of undefined (setting 'value')
    at http://localhost:1337/src_bootstrap_tsx.js:26717:53
    at Array.forEach (<anonymous>)
    at StepsService.updateStepParameters (http://localhost:1337/src_bootstrap_tsx.js:26715:39)
    at Object.<anonymous> (http://localhost:1337/src_bootstrap_tsx.js:25210:22)
    at invokeFunc (http://localhost:1337/node_modules_lodash_debounce_index_js.js:168:19)
    at trailingEdge (http://localhost:1337/node_modules_lodash_debounce_index_js.js:215:14)
    at timerExpired (http://localhost:1337/node_modules_lodash_debounce_index_js.js:203:14)
ERROR
Cannot set properties of undefined (setting 'value')
TypeError: Cannot set properties of undefined (setting 'value')
    at http://localhost:1337/src_bootstrap_tsx.js:26717:53
    at Array.forEach (<anonymous>)
    at StepsService.updateStepParameters (http://localhost:1337/src_bootstrap_tsx.js:26715:39)
    at Object.<anonymous> (http://localhost:1337/src_bootstrap_tsx.js:25210:22)
    at invokeFunc (http://localhost:1337/node_modules_lodash_debounce_index_js.js:168:19)
    at trailingEdge (http://localhost:1337/node_modules_lodash_debounce_index_js.js:215:14)
    at timerExpired (http://localhost:1337/node_modules_lodash_debounce_index_js.js:203:14)
    ```

**Comment by Delawen** [comment]
Closed as duplicated of #1561
